libs/cardano-ledger-api/testlib/Test/Cardano/Ledger/Api/DebugTools.hs

```diff
@@ -0,0 +1,19 @@
+{-# LANGUAGE TypeApplications #-}
+
+module Test.Cardano.Ledger.Api.DebugTools where
+
+import Cardano.Ledger.Binary (DecCBOR, EncCBOR, decodeFull', showDecoderError)
+import Cardano.Ledger.Binary.Encoding (serialize')
+import Cardano.Ledger.Conway (Conway)
+import Cardano.Ledger.Core (eraProtVerLow)
+import qualified Data.ByteString as BS
+
+readCBORFromFile :: DecCBOR a => FilePath -> IO a
+readCBORFromFile path = do
+  dat <- BS.readFile path
+  case decodeFull' (eraProtVerLow @Conway) dat of
+    Right x -> pure x
+    Left err -> error $ showDecoderError err
+
+writeCBORFromFile :: EncCBOR a => FilePath -> a -> IO ()
+writeCBORFromFile path val = BS.writeFile path $ serialize' (eraProtVerLow @Conway) val
```

Comment:
I think `FromFile` suffix is unnecessary

We need ability to supply version, we can;t expect these functions to be used just for Conway.

Use `MonadIO`

It could be useful to have two functions: with ability of supplying either the version or the era for types that are parametrized on era: 

```suggestion
readCBOR :: (DecCBOR a, MonadIO m) => Version -> FilePath -> m a
readCBOR version path = liftIO $ do
  dat <- BS.readFile path
  case decodeFull' version dat of
    Right x -> pure x
    Left err -> throwIO err

writeCBOR :: (EncCBOR a, MonadIO m) => Version -> FilePath -> a -> m ()
writeCBOR path = liftIO . BS.writeFile path . serialize' version

readEraCBOR :: forall t era m . (Era era, DecCBOR (t era), MonadIO m) => FilePath -> m (t era)
readEraCBOR = readEraCBOR (eraProtVerLow @era)

writeEraCBOR :: forall era t. (Era era, EncCBOR (t era), MonadIO m) => FilePath -> t era -> m ()
writeEraCBOR = writeCBOR (eraProtVerLow @era)
```